tests/integration/targets/rabbitmq_exchange/tasks/main.yml !

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
It looks like this file is an accidental copy. Please delete.

Reply:
Done.